Resources/Prototypes/Datasets/tips.yml

```diff
@@ -93,7 +93,7 @@
   - "The station's nuke is invincible. Go find the disk instead of trying to destroy it."
   - "Maintenance is full of equipment that is randomized every round. Look around and see if anything is worth using."
   - "We were all new once, be patient and guide new players, especially those playing intern roles, in the right direction."
-  - "Firesuits, winter coats and emergency hardsuits offer mild protection from the cold, allowing you to spend longer periods of time near breaches and space than if wearing nothing at all."
+  - "Firesuits, winter coats and emergency EVA offer mild protection from the cold, allowing you to spend longer periods of time near breaches and space than if wearing nothing at all."
```

Comment:
shouldn't this be: Firesuits, winter coats and emergency EVA suits ...

Reply:
yea damn